src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
+            </div>
+          </div>
+        </div>
+        <div class="chi-backdrop -closed">
+          <div class="chi-backdrop__wrapper">
+            <div class="chi-drawer -right -menu -position--absolute" id="drawer-2">
+              <div class="chi-drawer__header">
+                <span class="chi-drawer__title">Actions</span>
+                <button class="chi-button -icon -close" aria-label="Close">
                   <div class="chi-button__content">
-                    <i class="chi-icon icon-filter"></i>
+                    <i class="chi-icon icon-x"></i>
                   </div>
                 </button>
               </div>
+              <div class="chi-drawer__content -px--2">
+                <ul class="-no-style">
+                  <li class="-text--md">Download data</li>
+                  <div class="chi-divider"></div>
+                  <li class="-text--md">Refresh results</li>
+                  <div class="chi-divider"></div>
+                </ul>
+              </div>
             </div>
           </div>
-          <div class="chi-toolbar__end">
-            <div class="chi-toolbar__actions">
-              <div class="chi-toolbar__actions-desktop">
-                <button class="chi-button -icon -flat" id="button-br-download" aria-label="Download" data-tooltip="Download" data-position="top">
-                  <div class="chi-button__content">
-                    <i class="chi-icon icon-arrow-to-bottom"></i>
+        </div>
+
+        <!-- Javascript -->
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-br1'));
+          chi.drawer(document.getElementById('drawer-trigger-br2'));
+          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
+        </script>
+
+.-portal--show
+  h3 Portal base
+  .example.-mb--3
+    .-p--3
+      .chi-toolbar.-portal
+        .chi-toolbar__header
+          .chi-toolbar__start
+            .chi-toolbar__search
+              .chi-form__item
+                chi-search-input(portal size="lg" id="example__base-3")
+            .chi-divider.-vertical
+            .chi-toolbar__filters
+              .chi-toolbar__filters-desktop
+                .chi-form__item
+                  select(class="chi-select -lg -portal", id="example-ba5")
+                    option(value="") Select
+                    option Option 1
+                    option Option 2
+                    option Option 3
+                .chi-form__item
+                  select(class="chi-select -lg -portal", id="example-ba6")
+                    option(value="") Select
+                    option Option 1
+                    option Option 2
+                    option Option 3
+                button.chi-button.-portal.-icon.-primary.-flat(id="button-p-filters" aria-label="Filters" data-tooltip="Filters" data-position="top")
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.